src/javascript/app/pages/user/account/statement/statement.ui.js

```diff
@@ -63,7 +63,12 @@ const StatementUI = (() => {
         ], columns, 'data');
         $statement_row.children('.credit').addClass(credit_debit_type);
         $statement_row.children('.date').addClass('pre');
-        $statement_row.children('.desc').html(`${statement_data.desc}<br>`);
+        $statement_row.children('.desc').html(`<span>${statement_data.desc}</span><br>`);
+
+        // add processing time popup for withdrawal
+        if (transaction.action_type === 'withdrawal') {
+            $statement_row.children('.desc').find('span').attr('data-balloon', transaction.withdrawal_details);
```

Comment:
What is `data-balloon`?

Reply:
Do we use `data-balloon` anywhere else in `binary-static`?

Reply:
Its an attribute that adds a popup to an element

Reply:
Found it :)